### DISCUSSION TITLE
Which xtbpath output is final result?

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [grimme-lab/xtb#738] [Q&A] [answered]
Dear everyone, 

I am new to use reaction path methods and especially with xtb. 
I was just wondering, when i run a **'xtb Start.xyz --path End.xyz --input path.inp'**, which output file contains the correct reaction path? I get a lot of outputs all called **xtbpath_0.xyz, xtbpath_1.xyz** ... etc. 
I am trying to automate some calculations, and i need to know which file to open up to show the reaction animation, when the computation is done. Also is there a way to only get the final result as an output?

Kind regards
Zan

**Comment by gorges97** [comment, marked as answer]
Dear Zan,

the final, refined reaction path is in the **xtbpath.xyz** file. However, the status of the reaction path finder in xtb is highly experimental and not really reliable. I would recommend to use a more robust method like NEB in ORCA. 

Kind regards
Johannes

**Reply by NanoscaleSimulations** [reply]
Dear Johannes

Thank you so much for clarifying this for me.
My current work requires very fast and very approximate computations as i am trying to use atomic simulations to explain basic chemistry to middle-grade school students. That is why i am trying the xtbpath method as it seems very fast (finished in a few seconds) and i just need a rough idea about energies in reactions. So far, NEB has been too slow for me (at least through ASE). But i will take a look at ORCA as i am open to trying new things.

Kind regards
